contracts/Property.sol

```diff
@@ -119,6 +125,36 @@ contract Property is IProperty, OwnableUpgradeable, ReentrancyGuardUpgradeable {
         emit NewPaymentReceiver(_addr);
     }
 
+    /**
+        @notice Update KYG status of the given booking ID
+        @dev    Caller must be OPERATOR
+        @param  _id booking ID
+        @param  _status new KYG status
+     */
+    function updateKygStatusById(uint256 _id, KygStatus _status) external {
+        require(_msgSender() == management.operator(), "OnlyOperator");
+        BookingInfo memory info = booking[_id];
+        require(info.guest != address(0), "BookingNotFound");
+        require(
+            info.balance > 0 || isInsuranceFeePending[_id],
+            "BookingAlreadyFinalized"
+        );
+
+        InsuranceInfo storage insuranceInfo = insurance[_id];
+        require(insuranceInfo.damageProtectionFee > 0, "InsuranceNotFound");
```

Comment:
@bruno-dtravel , according to this [docs](https://dtravelnetwork.atlassian.net/wiki/spaces/DDV/pages/365920257/Superhog+integration), the status of KYG will decide the logic of charging or refunding damage protection fee, so if the booking is already finalized (balance of zero and no pending fee) then there's nothing to do in the contract side, if the contract still allow BE changing this status, it can lead to some confused cases when we need to check or debug the logic flow from the booking data on chain because the final KYG status is not correct with the appropriate logic flow already processed. My idea is that if the booking is already finalized with a status then contract will leave the booking in that status forever.